sdk/storage/Azure.Storage.Queues/api/Azure.Storage.Queues.netstandard2.0.cs

```diff
@@ -40,24 +40,38 @@ public partial class QueueClient
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Storage.Queues.Models.QueueMessage[]>> ReceiveMessagesAsync() { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Storage.Queues.Models.QueueMessage[]>> ReceiveMessagesAsync(int? maxMessages = default(int?), System.TimeSpan? visibilityTimeout = default(System.TimeSpan?), System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Storage.Queues.Models.QueueMessage[]>> ReceiveMessagesAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.Storage.Queues.Models.SendReceipt> SendMessage(Azure.Core.BinaryData message) { throw null; }
+        public virtual Azure.Response<Azure.Storage.Queues.Models.SendReceipt> SendMessage(Azure.Core.BinaryData message, System.TimeSpan? visibilityTimeout = default(System.TimeSpan?), System.TimeSpan? timeToLive = default(System.TimeSpan?), System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.Storage.Queues.Models.SendReceipt> SendMessage(Azure.Core.BinaryData message, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual Azure.Response<Azure.Storage.Queues.Models.SendReceipt> SendMessage(string messageText) { throw null; }
         public virtual Azure.Response<Azure.Storage.Queues.Models.SendReceipt> SendMessage(string messageText, System.TimeSpan? visibilityTimeout = default(System.TimeSpan?), System.TimeSpan? timeToLive = default(System.TimeSpan?), System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual Azure.Response<Azure.Storage.Queues.Models.SendReceipt> SendMessage(string messageText, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
I wonder if we should remove the `string` overloads just for the sake of simplicity.  We could also talk about implicit casts to BinaryData from common types so you could pass a string without wrapping in a .ctor.

Reply:
I had this at some point but decided to remove attributes. Implicit cast would be nice but I would it show up somehow in IntelliSense?

Reply:
In Service Bus, we left in the constructor overload that takes a string even though we have one that takes BinaryData. IMO it might be a stretch to expect users to discover that string can be passed for a BinaryData parameter (if we added an implicit cast from string to BD), but maybe if the type becomes ubiquitous enough that will change. I also tend to think that BD is more helpful for reducing the clutter on output API as opposed to input.